.ci/.bump-go-release-version.yml

```
@@ -37,6 +37,13 @@ projects:
     enabled: true
     labels: dependency,backport-skip
     description: It requires the version to be bumped first in golang-crossbuild project, then a new release will be added to https://github.com/elastic/golang-crossbuild/releases. Otherwise it will fail until the docker images are available.
+  - repo: cloudbeat
+    script: .ci/bump-go-release-version.sh
+    branches:
+      - main
+    enabled: true
+    labels: dependency
+    reviewer: elastic/cloudbeat
```

Since the project `cloudbeat` doesn't support the `1.19` go release yet, maybe we could add the override go version setting.

```suggestion
    reviewer: elastic/cloudbeat
    overrideGoVersion: "1.18"
```